lib/run_loop/physical_device/life_cycle.rb

```diff
@@ -0,0 +1,260 @@
+module RunLoop
+  # @!visibility private
+  module PhysicalDevice
+
+    # Raised when installation fails.
+    class InstallError < RuntimeError; end
+
+    # Raised when uninstall fails.
+    class UninstallError < RuntimeError; end
+
+    # Controls the behavior of various life cycle commands.
+    #
+    # You can override these values if they do not work in your environment.
+    #
+    # For cucumber users, the best place to override would be in your
+    # features/support/env.rb.
+    #
+    # For example:
+    #
+    # RunLoop::PhysicalDevice::LifeCycle::DEFAULT_OPTIONS[:timeout] = 60
+    DEFAULT_OPTIONS = {
+      :install_timeout => RunLoop::Environment.ci? ? 120 : 30
```

Comment:
Yes.  Reports from users show that CI systems using ideviceinstaller take at least 2 times as long to install apps.